### ISSUE TITLE
Add validation for OTP

### ISSUE THREAD

**Issue by gandarain** [anday013/react-native-otp-entry#83]
Is there any way to add validation for OTP, to prevent user input space and special character.
Thank you.

**Comment by anday013** [comment]
@gandarain  yes, have you tried to set `type` prop from [Props](https://github.com/anday013/react-native-otp-entry?tab=readme-ov-file#props)?

**Comment by gandarain** [comment]
Thank you @anday013
